src/recipes/recipes.service.ts

```diff
@@ -34,4 +36,15 @@ export class RecipesService {
 
     return recipe;
   }
+
+  async addRecipe(
+    authorId: string,
+    addRecipeDto: AddRecipeDto,
+  ): Promise<GetRecipeResponse> {
+    try {
+      return await this.recipeRepository.addRecipe(authorId, addRecipeDto);
+    } catch (error) {
```

Comment:
and I see you are using it here in the service so this `await` keyword in repository is unnecessary